pkg/globalmanager/config/config.go

```diff
@@ -54,6 +55,9 @@ type ControllerConfig struct {
 
 	// lc config to info the worker
 	LC LCConfig `json:"localController,omitempty"`
+
+	// kb config to info the worker
+	KB KBConfig `json:"knowledgeBaseServer,omitempty"`
```

Comment:
inconsistent here: knowledgeBaseServer vs knowledgeBaseService in gm.yaml

Reply:
fix it